CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## [2.0.120240111]
+- Updating to latest B2K CLI version. Please refer for release notes [here](TBD)
```

Comment:
👍☕️ This TBD reside here mentioning so that folks know that the release user this PR has to go out first - https://github.com/Azure/Bridge-To-Kubernetes/pull/349

Reply:
updated this. Thank you.